CMakeLists.txt

```diff
@@ -505,7 +499,19 @@ if (HB_HAVE_ICU)
   target_link_libraries(harfbuzz-icu harfbuzz ${THIRD_PARTY_LIBS})
 
   if (BUILD_SHARED_LIBS)
-    set_target_properties(harfbuzz harfbuzz-icu PROPERTIES VISIBILITY_INLINES_HIDDEN TRUE)
+    set_target_properties(harfbuzz harfbuzz-icu PROPERTIES 
+      VISIBILITY_INLINES_HIDDEN TRUE
+      FRAMEWORK TRUE
+      FRAMEWORK_VERSION "${HB_VERSION}"
+      PUBLIC_HEADER "${project_headers}"
+      PRODUCT_BUNDLE_IDENTIFIER "harfbuzz.harfbuzz-icu"
+      XCODE_ATTRIBUTE_INSTALL_PATH "@rpath"
+      OUTPUT_NAME "harfbuzz-icu"
+      XCODE_ATTRIBUTE_CODE_SIGN_IDENTITY ""
+      MACOSX_FRAMEWORK_IDENTIFIER "harfbuzz-icu"
+      MACOSX_FRAMEWORK_SHORT_VERSION_STRING "${HB_VERSION}"
+      MACOSX_FRAMEWORK_BUNDLE_VERSION "${HB_VERSION}"
+    )
```

Comment:
This is now doing a framework build even though it does not check for `BUILD_FRAMEWORK`.

Reply:
Thanks for pointing it out. Will be fixed.